examples/src/bin/ech-client.rs

```diff
@@ -28,7 +28,7 @@
 //! "SSL_ECH_STATUS": "success"
 //! ```
 
-use std::error::Error;
+use core::error::Error;
 use std::fs;
```

Comment:
I think it would be nice to add a blank line here like this, throughout rustls-examples in this fork:
```diff
 use core::error::Error;
+
 use std::fs;
```

but in this case I think blank lines would be needed to separate some other blocks of imports.